src/call/Endpoint.js

```diff
@@ -111,6 +111,35 @@ export default class Endpoint {
         }
     }
 
+    /**
+     * Method for requests the specified video size for the video stream.
+     * @param {*} streamId 
+     * @param {*} width 
+     * @param {*} height 
+     * @returns 
```

Comment:
Please add complete documentation for new APIs

Don't forget to add `@memberOf Voximplant.Endpoint`

Reply:
done